roles/ceph-ansible/tasks/main.yml

```diff
@@ -0,0 +1,180 @@
+---
+- name: Create directory for keys
+  file: path=/root/ceph-ansible-keys state=directory
+
+- name: Copy ceph ansible files into different location so current user can change it
+  shell: cp -R /usr/share/ceph-ansible/ {{ home_dir }}
+
+- name: Copy sample files
+  copy:
+    src: "{{ home_dir }}/ceph-ansible/{{ item.src }}"
+    dest: "{{ home_dir }}/ceph-ansible/{{ item.dst }}"
+    remote_src: "True"
+  with_items:
+    - dst: "group_vars/all.yml"
+      src: "group_vars/all.sample"
+    - dst: "group_vars/mons.yml"
+      src: "group_vars/mons.sample"
+    - dst: "group_vars/osds.yml"
+      src: "group_vars/osds.sample"
+    - dst: "site.yml"
+      src: "site.yml.sample"
+
+# edit all.yml file
+
+- name: Set fetch directory
+  lineinfile:
+    dest: "{{ home_dir }}/ceph-ansible/group_vars/all.yml"
+    regexp: "^#fetch_directory: fetch/"
+    line: "fetch_directory: /root/ceph-ansible-keys"
+
+- name: Change default Ceph cluster name
+  lineinfile:
+    dest: "{{ home_dir }}/ceph-ansible/group_vars/all.yml"
+    regexp: '^#cluster: ceph.*'
+    line: 'cluster: test_name'
+
+- name: Set automated fsid generation
+  lineinfile:
+    dest: "{{ home_dir }}/ceph-ansible/group_vars/all.yml"
+    regexp: '^#generate_fsid: true.*'
+    line: 'generate_fsid: true'
+
+# Disabled because this list in directly in ceph-ansible
```

Comment:
Probably just typo?  _Disabled because this list **is** directly in ceph-ansible._